test/writer.xwiki

```diff
@@ -0,0 +1,862 @@
+This is a set of tests for pandoc. Most of them are adapted from John Gruber’s markdown test suite.
+
+
+
+----
+
+= Headers =
+
+== Level 2 with an [[embedded link>>/url]] ==
+
+=== Level 3 with //emphasis// ===
+
+==== Level 4 ====
+
+===== Level 5 =====
+
+= Level 1 =
+
+== Level 2 with //emphasis// ==
+
+=== Level 3 ===
+
+with no blank line
+
+
+== Level 2 ==
+
+with no blank line
+
+
+
+----
+
+= Paragraphs =
+
+Here’s a regular paragraph.
+
+
```

Comment:
There are two blank lines between each paragraph.  Is that needed in xwiki?  If not it should be avoided.

Note that Text.Pandoc.Pretty might be worth using instead of just generating Text.